src/interfaces/coral_web/src/components/Shared/Dropdown/Dropdown.tsx

```diff
@@ -141,7 +143,8 @@ export const Dropdown: React.FC<Props> = ({
                 {
                   'px-3 py-4': kind === 'default' || !label,
                   'px-3 pb-2.5 pt-7': kind === 'cell' && label,
-                }
+                },
+                buttonClassName
```

Comment:
needed to add this to the end to override the padding styles-- can omit this altogether though if we are cool having a thicker nav bar

would look like:
![Screenshot 2024-04-23 at 16 13 08](https://github.com/cohere-ai/cohere-toolkit/assets/140425731/31ad7257-ec5d-431c-a082-d82c63196335)
![Screenshot 2024-04-23 at 16 12 59](https://github.com/cohere-ai/cohere-toolkit/assets/140425731/5bddf272-8a4a-4360-b78d-1409cdc18176)